From 64b806ab702a6f445fd84aca5f03bb85b86a66e3 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 11 Nov 2021 18:37:09 -0800
Subject: [PATCH] framework: do not mix plain and keyword
 target_link_libraries() signatures

Fixes
| CMake Error at framework/CMakeLists.txt:461 (target_link_libraries):
|   The plain signature for target_link_libraries has already been used with
|   the target "framework".  All uses of target_link_libraries with a target                                            |   must be either all-keyword or all-plain.
|
|   The uses of the plain signature are here:
|                                                                                                                       |    * framework/CMakeLists.txt:453 (target_link_libraries)
|

Upstream-Status: Submitted [https://github.com/KhronosGroup/Vulkan-Samples/pull/395]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 framework/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/framework/CMakeLists.txt b/framework/CMakeLists.txt
index 948c796..13d3324 100644
--- a/framework/CMakeLists.txt
+++ b/framework/CMakeLists.txt
@@ -450,7 +450,7 @@ target_link_libraries(${PROJECT_NAME} PUBLIC
     plugins)
 
 if(${NEED_LINK_ATOMIC})
-    target_link_libraries(${PROJECT_NAME} atomic)
+    target_link_libraries(${PROJECT_NAME} PUBLIC atomic)
 endif()
 
 # Link platform specific libraries
-- 
2.33.1

